src/main/java/model/Fruit.java

```diff
@@ -0,0 +1,23 @@
+package model;
+
+public class Fruit {
```

Comment:
You can delete this class. **quantity** is not a Fruit property. quantity is Storage property.